openassessment/xblock/static/js/spec/lms/oa_base.js

```diff
@@ -50,21 +50,17 @@ describe("OpenAssessment.BaseView", function() {
     Args:
         callback (function): Function that takes no arguments.
     **/
-    var loadSubviews = function(callback) {
-        runs(function() {
-            view.load();
-        });
-
-        waitsFor(function() {
-            return !$(".openassessment__steps__step").hasClass('is--loading');
-        });
-
-        runs(function() {
-            return callback();
-        });
+    var loadSubviews = function(done) {
+        view.load();
+
+        // Wait for the is--loading class to have been removed
+        setTimeout(function() {
+            expect($(".openassessment__steps__step").hasClass('is--loading')).toBeFalsy();
+            done(); // call this to finish off the it block
+        }, 500);
```

Comment:
@jzoldak I removed the timeout and changed the code to execute synchronously and it seems fine.